workloads/scale-perf/README.md

```diff
@@ -53,24 +60,59 @@ Enable/Disable collection of metadata
 
 ### COMPARE (not implemented yet)
 Default: `false`   
+
 Enable/Disable the ability to compare two runs. If set to `true`, the next set of environment variables pertaining to the type of test are required
 
 ### BASELINE_CLOUD_NAME
-Default: ``    
+Default: ''
+
 Name you would like to give your baseline cloud. It will appear as a header in the CSV file
 
-### ES_SERVER_BASELINE 
+### ES_SERVER_BASELINE
 Default: `https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443`
-Elasticsearch server used used by the baseline run 
+
+Elasticsearch server used used by the baseline run
 
 ### BASELINE_UUID
-Default: ``   
-Baseline UUID 
+Default: ''
+
+Baseline UUID
 
 ### CERBERUS_URL
-Default: ``     
+Default: ''
+
 URL to check the health of the cluster using Cerberus (https://github.com/cloud-bulldozer/cerberus).
 
+## ROSA Integration
+Default: ''
+
+When scaling clusters installed on ROSA, following variables are also required:
+
+### ROSA_CLUSTER_NAME
+Default: ''
+
+Cluster Name as it is shown when running `rosa list clusters`
+
+### ROSA_ENVIRONMENT
+Default: ''
+
+Environment where cluster is deployed (Production, Staging)
+
+### ROSA_TOKEN
+Default: ''
+
+Token used to login on ROSA
+
+### AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY
```

Comment:
Extremely minor nit but can you separate these 2 into separate items? I had to re-read it 3 times before i realized how you wanted the key and id lol.

Reply:
done